node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -167,6 +175,7 @@ impl ChainScraper {
 		// Clean up finalized:
 		for finalized_candidate in finalized.into_values().flatten() {
 			self.included_candidates.remove(&finalized_candidate);
+			self.backed_candidates.remove(&finalized_candidate);
 		}
 	}
 
```

Comment:
May want to change the comment three lines below to say we keep track of backed candidates too.